pkgs/by-name/ti/tiramisu/package.nix

```
@@ -7,16 +7,14 @@
   vala,
 }:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "tiramisu";
-  # FIXME: once a newer release in upstream is available
-  version = "2.0-unstable-2023-03-29";
+  version = "2.0.20240610";
 
   src = fetchFromGitHub {
     owner = "Sweets";
-    repo = "tiramisu";
-    # FIXME: use the current HEAD commit as upstream has no releases since 2021
-    rev = "5dddd83abd695bfa15640047a97a08ff0a8d9f9b";
+    repo = finalAttrs.pname;
```

Please don't use `pname`, this is considered an antipattern
```suggestion
    repo = "tiramisu";
```